pkg/webui/account/store/actions/user.js

```diff
@@ -18,7 +18,11 @@ export const GET_USER_BASE = 'GET_USER'
 export const [
   { request: GET_USER, success: GET_USER_SUCCESS, failure: GET_USER_FAILURE },
   { request: getUser, success: getUserSuccess, failure: getUserFailure },
-] = createRequestActions(GET_USER_BASE, id => ({ id }), (id, selector) => ({ selector }))
+] = createRequestActions(
+  GET_USER_BASE,
+  id => ({ id }),
+  (id, selector) => ({ selector }),
+)
```

Comment:
The changes that result from updating prettier should not be part of this PR, but rather be incorporated in the PR that updates prettier. This should fix itself when you rebase this on master after #4083 was merged.